content/400-pulse/600-faq.mdx

```diff
@@ -123,4 +123,22 @@ If you encounter issues with SSL/TLS encryption while using Pulse, ensure your d
 postgres://username:password@hostname:port/database?sslmode=require
 ```
 
-This [setting](/orm/overview/databases/postgresql#configuring-an-ssl-connection) will enforce SSL/TLS encryption and accept self-signed certificates.
\ No newline at end of file
+This [setting](/orm/overview/databases/postgresql#configuring-an-ssl-connection) will enforce SSL/TLS encryption and accept self-signed certificates.
+
+## When should I enable static IP for Prisma Pulse?
+
+A static IP address is an IPv4 or an IPv6 address that is fixed. Unlike dynamic IP addresses, which can change unpredictably, traffic from static IP addresses can be easily identified. 
+
+![What is a static IP](./../300-accelerate/images/static-ip.png)
```

Comment:
I don't think this info belongs here. Should we have another "what is a static IP" section?